modules/tensor_mechanics/src/materials/ComputeReturnMappingStress.C

```diff
@@ -47,13 +54,22 @@ ComputeReturnMappingStress::ComputeReturnMappingStress(const InputParameters & p
     _relative_tolerance(parameters.get<Real>("relative_tolerance")),
     _absolute_tolerance(parameters.get<Real>("absolute_tolerance")),
     _output_iteration_info(getParam<bool>("output_iteration_info")),
+    _perform_finite_strain_rotations(getParam<bool>("perform_finite_strain_rotations")),
     _elasticity_tensor(getMaterialPropertyByName<RankFourTensor>(_base_name + "elasticity_tensor")),
     _elastic_strain_old(declarePropertyOld<RankTwoTensor>(_base_name + "elastic_strain")),
-    _strain_increment(getMaterialProperty<RankTwoTensor>(_base_name + "strain_increment"))
+    _strain_increment(getMaterialProperty<RankTwoTensor>(_base_name + "strain_increment")),
+    _plastic_strain(declareProperty<RankTwoTensor>(_base_name + "plastic_strain")),
```

Comment:
Hold on -- I was mixed up.  This is the class that drives multiple plasticity or creep models, so `_plastic_strain` here contains a combination of the inelastic strains from all of the models that are being driven.  It seems to me like you might want every model to create its own `plastic_strain` or `creep_strain` property in RecomputeRadialReturn, but this one should really have a generic name like `inelastic_strain`.

By the way, I think this class is poorly named, and I've gotten confused like this multiple times. The name `ComputeReturnMappingStress` doesn't convey that this drives multiple models, and the models that it drives don't really even necessarily have to use return mapping algorithms.  I don't have a great alternative to propose at the top of my head, so we'll save renaming it for another time.